Drop-In/src/main/res/anim/bt_bottom_sheet_layout_animation.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layoutAnimation xmlns:android="http://schemas.android.com/apk/res/android"
-    android:delay="@integer/bt_activity_transition_duration"
+    android:delay="@integer/fragment_transition_duration"
```

Comment:
Let's prefix this with `bt` for consistency

Reply:
Yeah got ahead of myself I want to remove those lol.

Reply:
Done!

Reply:
Wait nah good call. It turns out these help prevent collisions with merchant apps https://stackoverflow.com/a/33247312